### ISSUE TITLE
Scrolling away from hover-triggered popover doesn't cause popover to hide

### ISSUE THREAD

**Issue by InternSix** [twbs/bootstrap#19366]
If user hover over disabled button with popover set to hover and then scrolls while still hovering over the button, the popover will stay visible. Ideally popover would hide on scroll.

Browser Used: Chrome Version 48.0.2564.116 m
Bootstrap v3.3.6

See example:
http://www.bootply.com/NGklfSC8T5

**Comment by twbs-lmvtfy** [comment]
Hi @InternSix!

You appear to have posted a live example (http://s.bootply.com/render/gR8IWleZYp), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 48, column 200: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by chief10** [comment]
Just wondering, why does javascript need to be involved in this at all? Why not just have the popover have a display of "hidden" until its parent element is hovered over?

**Comment by udiboy1209** [comment]
This is being caused because the button is disabled, so the hover out event never fires on it when you scroll away. Maybe popovers should be disabled on disabled elements, but I'm just saying that because I can't see a workaround which wouldn't break normal functionality :stuck_out_tongue:

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
